entrypoint.sh

```diff
@@ -14,7 +14,7 @@ fi
 # Pick a random port above 32768
 DEFAULT_PORT=$RANDOM
 let "DEFAULT_PORT += 32768"
-echo [INFO] Tunneling ${SSH_HOSTUSER:=root}@${SSH_HOSTNAME:=localhost}:${SSH_TUNNEL_REMOTE:=${DEFAULT_PORT}} to ${SSH_TUNNEL_HOST=localhost}:${SSH_TUNNEL_LOCAL:=22}
+echo [INFO] Tunneling over ${SSH_HOSTUSER:=root}@${SSH_HOSTNAME:=localhost} from ${SSH_BIND_IP:=0.0.0.0}:${SSH_TUNNEL_REMOTE:=${DEFAULT_PORT}} to ${SSH_TUNNEL_HOST=localhost}:${SSH_TUNNEL_LOCAL:=22}
```

Comment:
In colloquial English, the words `over` and `from` change this sentence to mean the opposite of what it should.

The SSH_HOSTUSER@SSH_HOSTNAME is being tunneled OVER (more appropriately "inside") the REMOTE.

Please revert the words.